programs/survival/auto_lighter.sc

```diff
@@ -27,10 +28,12 @@ __distance_sq(vec1, vec2) -> reduce(vec1 - vec2, _a + _*_, 0);
 
 global_effect_radius = 128;
 
+global_survival=false;
+
 spread_torches(player) ->
 (
 	if (global_spread_love && player~'holds':0 == 'torch',
```

Comment:
```suggestion
	if (global_spread_love && player~'holds':0 == 'torch' && player ~'gamemode_id' == initial_gamemode,
```

Reply:
cannot add suggestions to the code below, just add initial_gamemode to the schedules for spread_torches